### PR TITLE
Back, left, and right InfraRed sensors

### PR DESCRIPTION
**Pull request by Akuien**
Implementation for the left, back and right infrared sensors to detect obstacle.

### :ballot_box_with_check: Definition of Done checklist
- [x] Meaningful title and valuable description in pull request
- [x] Code change tested manually if not covered by automated tests
- [x] Pull request linked to one or more issues/stories
- [x] Acceptance criteria of the linked stories satisfied
- [x] Code change comprehended by those who approved it
- [x] State of target branch maintained or improved after the code change

### CODE REVIEW

File: CarEshipApp/app/src/main/java/com/example/careshipapp/user_management/DBHelperClass.java [hunk 1]
```diff
@@ -8,6 +8,9 @@
 
 import androidx.annotation.Nullable;
 
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
```
Comment (d0nate110) [inline CarEshipApp/app/src/main/java/com/example/careshipapp/user_management/DBHelperClass.java:12]: Might have added these by mistake?

**Review by d0nate110 — CHANGES_REQUESTED** [review]
You should remove lines 11 and 12, since they are not needed.

File: CarEshipApp/app/src/main/java/com/example/careshipapp/user_management/DBHelperClass.java (hunk 1 above)
Comment (Akuien) [reply CarEshipApp/app/src/main/java/com/example/careshipapp/user_management/DBHelperClass.java:12]: Ooh its great you spotted those. Will remove the right away.

**Review by d0nate110 — APPROVED** [review]
Tested the code, the sensors are working as they should, receiving distance data up to 40 cm away from the car. Code changes approved.

**Pull request merged**